bip-andychase.mediawiki

```diff
@@ -0,0 +1,116 @@
+* If an organization or community does not meet that requirement, it should conglomerate itself with other communities and organizations so that it does.
+* The segments that committees can be based around are:
+** Bitcoin software
+** Exchanges/Merchants/services/payment processors
+** Mining operators
+** User communities
+* A person may be represented by any number of segments, but a committee cannot re-use the same resource as another committee in the same segment.
+
+'''Committee Declarations.''' 
+* At any point, a Committee Declaration can be posted.
+* This Declaration must contain details about:
+** The segment the Committee is representing
+** Who the committee claim to represent and it's compositional makeup (if made up of multiple miner orgs, user orgs, companies, clients, etc).
+** Proof of claim and minimum 1% stake via:
+*** Software: proof of ownership and user base (Min 1% of Bitcoin userbase)
+*** Merchant: proof of economic activity (Min 1% of Bitcoin economic activity)
+*** Mining: proof of work (Min 1% of Hashpower)
+*** For a user organization, auditable signatures qualifies for a valid committee (Min 1% of Bitcoin userbase)
+** Who is running the committee, their names and roles
+** How represented members can submit comments to the committee
+** A code of conduct and code of ethics which the committee promises to abide by
+* A committee declaration is accepted if:
+** The declaration includes all of the required elements
+** The stake is considered valid
+** Committee validation is considered when considering the results of opinions submitted by committee on a BIP. A committee must have met the required stake percentage before a BIP is submitted for comments, and have maintained that stake until a valid opinion is submitted.
+* Committees can dissolve at any time or submit a declaration at any time
+* Declaration must have been submitted no later than the third day following a BIP's request for comments to be eligible for inclusion in a BIP
+
+== BIP Process Management Role ==
+
+BIPs, Opinions, and Committee Declaration must be public at all times.
+
+A BIP Process Manager should be chosen who is in charge of:
+
+* Declaring where and how BIPs, Opinions, and Committee Declaration should be posted and updated officially.
+* Maintaining the security and authenticity of BIPs, Opinions, and Committee Declarations
+* Publishing advisory documents about what kinds of proof of stakes are valid and what kinds should be rejected.
+* Naming a series of successors for the roles of the BIP Process Manager and BIP Editor (BIP-001) as needed.
+
+== Conditions for activation ==
+
+In order for this process BIP to become active, it must succeed by its own rules. At least a 4% sample of the Bitcoin community must be represented, with at least one committee in each segment included. Once at least one committee has submitted a declaration, a request for comments will be called and the process should be completed from there.
```

Comment:
Until this BIP is active, its rules do not apply, so this would be a form of circular reasoning. I like the idea of putting conditions for activation in the BIP text, but I don't think we can just let the author set any conditions they like either...